cluster-up/cluster/kind/check-cluster-up.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+
+set -exuo pipefail
+
+SCRIPT_PATH=$(dirname "$(realpath "$0")")
+
+kubevirtci_path="$(realpath "${SCRIPT_PATH}/../../..")/"
+PROVIDER_PATH="${kubevirtci_path}/cluster-up/cluster/${KUBEVIRT_PROVIDER}"
+
+RUN_KUBEVIRT_CONFORMANCE=${RUN_KUBEVIRT_CONFORMANCE:-"false"}
+
+(
+  cd $kubevirtci_path
+  kubectl="./cluster-up/kubectl.sh"
+  echo "Wait for pods to be ready.."
+  timeout 5m bash -c "until ${kubectl} wait --for=condition=Ready pod --timeout=30s --all  -A; do sleep 1; done"
+  timeout 5m bash -c "until ${kubectl} wait --for=condition=Ready pod --timeout=30s -n kube-system --all; do sleep 1; done"
+  ${kubectl} get nodes
+  ${kubectl} get pods -A
+  echo ""
+  
+  nodes=$(${kubectl} get nodes --no-headers | awk '{print $1}')
+  for node in $nodes; do
```

Comment:
some thing nice is to create an array of command strings and then iterate on it , executing it,
wdyt?

Reply:
I agree its nicer, I didnt do it because I wanted to keep the log messages before each command.
Putting it in bash equivalent for map doesn't make things prettier in this case.

Reply:
You can still keep log messages or whatever you desire in the loop isnt it ?
something like this wouldn't be nicer?
```
declare -a arr=("element1" "element2" "element3")
for i in "${arr[@]}"
do
   echo "$i" # exec here instead
done
```

Reply:
> You can still keep log messages or whatever you desire in the loop isnt it ?

It will print the command we executes and not its descriptions, for example
for `ip a` command, it will print 
`[sriov-worker] ip r` 
instead of:
`network interfaces status:`

It makes the log easier to follow, cosmetics.